<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>840</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabBar" name="tabBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PreviewWidget" name="previewWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>Recent</string>
     </property>
    </widget>
    <addaction name="actionNew_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSaveDocument"/>
    <addaction name="actionSaveAllDocuments"/>
    <addaction name="separator"/>
    <addaction name="actionClose_project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFontManager"/>
    <addaction name="actionLocalizationManager"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolbar">
   <property name="windowTitle">
    <string>Edit Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_project"/>
   <addaction name="actionOpen_project"/>
   <addaction name="actionSaveDocument"/>
   <addaction name="actionSaveAllDocuments"/>
   <addaction name="actionClose_project"/>
   <addaction name="separator"/>
   <addaction name="actionPixelized"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="PropertiesWidget" name="propertiesWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>250</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="FileSystemDockWidget" name="fileSystemDockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>250</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>File System</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="PackageWidget" name="packageWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>250</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Package</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="LibraryWidget" name="libraryWidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>250</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="ConsoleDockWidget" name="consoleDockWidget">
   <property name="windowTitle">
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
  </widget>
  <action name="actionOpen_project">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/openproject.png</normaloff>:/Icons/openproject.png</iconset>
   </property>
   <property name="text">
    <string>Open project</string>
   </property>
   <property name="toolTip">
    <string>Open project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/newscene.png</normaloff>:/Icons/newscene.png</iconset>
   </property>
   <property name="text">
    <string>New project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionHierarchy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hierarchy tree</string>
   </property>
  </action>
  <action name="actionLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Library</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionFontManager">
   <property name="text">
    <string>Font Manager</string>
   </property>
  </action>
  <action name="actionClose_project">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/101.png</normaloff>:/Icons/101.png</iconset>
   </property>
   <property name="text">
    <string>Close project</string>
   </property>
   <property name="toolTip">
    <string>Close project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionLocalizationManager">
   <property name="text">
    <string>Localization Manager</string>
   </property>
  </action>
  <action name="actionSaveDocument">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/savescene.png</normaloff>:/Icons/savescene.png</iconset>
   </property>
   <property name="text">
    <string>Save document</string>
   </property>
   <property name="toolTip">
    <string>Save changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>UI Editor Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/118.png</normaloff>:/Icons/118.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/117.png</normaloff>:/Icons/117.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSaveAllDocuments">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/savesceneall.png</normaloff>:/Icons/savesceneall.png</iconset>
   </property>
   <property name="text">
    <string>Save all documents</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRepack_And_Reload">
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/reloadtextures.png</normaloff>:/Icons/reloadtextures.png</iconset>
   </property>
   <property name="text">
    <string>Repack And Reload</string>
   </property>
   <property name="toolTip">
    <string>Repack and reload sprites</string>
   </property>
  </action>
  <action name="actionPixelized">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Data/icons.qrc">
     <normaloff>:/Icons/pixelated.png</normaloff>:/Icons/pixelated.png</iconset>
   </property>
   <property name="text">
    <string>Pixelized</string>
   </property>
   <property name="toolTip">
    <string>Pixelized/Smooth Graphics</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LibraryWidget</class>
   <extends>QDockWidget</extends>
   <header>UI/Library/LibraryWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PackageWidget</class>
   <extends>QDockWidget</extends>
   <header>UI/Package/PackageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>qtabbar.h</header>
  </customwidget>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>UI/Preview/PreviewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>PropertiesWidget</class>
   <extends>QDockWidget</extends>
   <header>UI/Properties/PropertiesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConsoleDockWidget</class>
   <extends>QDockWidget</extends>
   <header>UI/ConsoleView/ConsoleDockWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileSystemDockWidget</class>
   <extends>QDockWidget</extends>
   <header>UI/FileSystemView/FileSystemDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
